frontend/lib/admin.ts

```diff
@@ -88,26 +88,17 @@ export class Admin {
    * @throws {@link ApiError}
    * @throws {@link OfflineError}
    *
-   * @param {string} password - validate password
-   * @param {string} passwordConfirmation - MUST match passwordConfirmation
+   * @param {string} email - email of admin account
+   * @param {string} name - updated name
    * @param {string} token - JSON Web Token
    * @returns {Promise<void>}
    */
-  async update(
-    password: string,
-    passwordConfirmation: string,
+  static async update(
+    email: string,
+    name: string,
```

Comment:
Why make this `static` when you already have an array of `Admin` or a singular `Admin`?

Reply:
Because on edit only has name, email, token. Doesn't have an Admin object. All of the other Admin functions are static
![Screen Shot 2020-12-01 at 1 46 06 PM](https://user-images.githubusercontent.com/38922051/100783304-9a07f800-33db-11eb-9618-684e4af295e4.png)

Reply:
Ah okay, maybe this should cause a redesign of the frontend Api Admin.